pages/techgenius/tg-planet-user-manual.md

```diff
@@ -497,10 +460,17 @@ In order to update software and receive Publications,
     **Login:** Admin User Name
     **Password:** Admin Password
 
-  2. After logging in, any updates and/or available Publications will be visible as shown below.
-
-   ![Dashboard Software Update](images/tg-software-overview-update.png)
-
-  3. These two gray buttons in red writing will allow you to update the software and sync Publications. First, click “Update Available (0.12.33)”.
-
-  4. The update will automatically run and may take several minutes depending on your internet connection. While the loading screen is still spinning, remain connected and wait until you get a message notifying you that the software update has been successful.
+  2. Checking your software by selecting the manager setting from the top right corner.
+  
+  ![Software check](images/tg-software-update.png)
+  
+  3. As shown below, showing software of Nation Version and Local Version.
+  
+  ![Software version](images/tg-software-update3.png)
+  
+  4. **Upgrade planet apk** will allow you to update the software and sync Publications.
+  
+  ![Software update](images/tg-software-update2.png)
+   
+
+  5. The update will run and may take several minutes depending on your internet connection. While the loading screen is still spinning, remain connected and wait until you get a message notifying you that the software update has been successful.
```

Comment:
There is verify button shown once update is done successfully. So you can verify if upgrade was successful or had some error